bundles/org.openhab.ui/web/src/assets/semantics.js

```diff
@@ -1,5 +1,5 @@
 
 export const Locations = [ 'Location', 'Indoor', 'Apartment', 'Building', 'Garage', 'House', 'Shed', 'SummerHouse', 'Floor', 'GroundFloor', 'FirstFloor', 'SecondFloor', 'ThirdFloor', 'Attic', 'Basement', 'Corridor', 'Room', 'Bathroom', 'Bedroom', 'BoilerRoom', 'Cellar', 'DiningRoom', 'Entry', 'FamilyRoom', 'GuestRoom', 'Kitchen', 'LaundryRoom', 'LivingRoom', 'Office', 'Veranda', 'Outdoor', 'Carport', 'Driveway', 'Garden', 'Patio', 'Porch', 'Terrace' ]
-export const Equipment = [ 'Equipment', 'AlarmSystem', 'BackDoor', 'Battery', 'Blinds', 'Boiler', 'Camera', 'Car', 'CeilingFan', 'CellarDoor', 'CleaningRobot', 'Dishwasher', 'Door', 'Doorbell', 'Dryer', 'Fan', 'Freezer', 'FrontDoor', 'GarageDoor', 'Gate', 'HVAC', 'KitchenHood', 'InnerDoor', 'Inverter', 'LawnMower', 'Lightbulb', 'LightStripe', 'Lock', 'MotionDetector', 'NetworkAppliance', 'Oven', 'PowerOutlet', 'Projector', 'Pump', 'RadiatorControl', 'Receiver', 'Refrigerator', 'RemoteControl', 'Screen', 'Sensor', 'SideDoor', 'Siren', 'Smartphone', 'SmokeDetector', 'Speaker', 'Television', 'Valve', 'VoiceAssistant', 'WallSwitch', 'WashingMachine', 'WebService', 'WeatherService', 'WhiteGood', 'Window' ]
+export const Equipments = [ 'Equipment', 'AlarmSystem', 'BackDoor', 'Battery', 'Blinds', 'Boiler', 'Camera', 'Car', 'CeilingFan', 'CellarDoor', 'CleaningRobot', 'Dishwasher', 'Door', 'Doorbell', 'Dryer', 'Fan', 'Freezer', 'FrontDoor', 'GarageDoor', 'Gate', 'HVAC', 'KitchenHood', 'InnerDoor', 'Inverter', 'LawnMower', 'Lightbulb', 'LightStripe', 'Lock', 'MotionDetector', 'NetworkAppliance', 'Oven', 'PowerOutlet', 'Projector', 'Pump', 'RadiatorControl', 'Receiver', 'Refrigerator', 'RemoteControl', 'Screen', 'Sensor', 'SideDoor', 'Siren', 'Smartphone', 'SmokeDetector', 'Speaker', 'Television', 'Valve', 'VoiceAssistant', 'WallSwitch', 'WashingMachine', 'WebService', 'WeatherService', 'WhiteGood', 'Window' ]
```

Comment:
"Equipments" is not proper English, see #438 :)
So this should remain as it is (same with other occurrences below).

Reply:
Ah dang, I thought it was just an inconsistency. Fixed it.